backend/src/main/java/com/google/capstone/TextToSpeech.java

```diff
@@ -0,0 +1,64 @@
+package com.google.capstone;
+
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.cloud.texttospeech.v1.AudioConfig;
+import com.google.cloud.texttospeech.v1.AudioEncoding;
+import com.google.cloud.texttospeech.v1.SsmlVoiceGender;
+import com.google.cloud.texttospeech.v1.SynthesisInput;
+import com.google.cloud.texttospeech.v1.SynthesizeSpeechResponse;
+import com.google.cloud.texttospeech.v1.TextToSpeechClient;
+import com.google.cloud.texttospeech.v1.VoiceSelectionParams;
+import com.google.protobuf.ByteString;
+import java.io.FileOutputStream;
+import java.io.OutputStream;
+
+
+
+@WebServlet(name = "TextToSpeech", value = "/api/v1/tts")
+public class TextToSpeech extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String textString = request.getParameter("text");
+    if (!textString.isEmpty()) {
+      try (TextToSpeechClient textToSpeechClient = TextToSpeechClient.create()) {
+        ByteString audioContents = generateAudio(textToSpeechClient, textString);
+        createFile(audioContents, response);
```

Comment:
Do we ever delete the mp3 files? I'm guessing not because you just overwrite the same file every time, so it's not a big deal right now... but see my comment below about using the same filename every time.